doc/tingle/tingle.js

```diff
@@ -120,14 +120,25 @@
             if(!close) return;
         }
 
-        this.modal.style.display = 'none';
```

Comment:
We can't only rely on ```visibility: hidden``` and ```opacity: 0``` to hide the modal. During tingle development, I've encountered a problem (I don't remember exactly the bug) and have come to the conclusion that ```display: none``` was mandatory.